Preface/Preface.pillar

```diff
@@ -83,16 +83,14 @@ that you may find useful:
 - *http://www.pharo.org>http://www.pharo.org* is the main web site of Pharo.
 - On IRC, you can find us on the freenode.net server, channel "pharo".
 - You can also ask questions on Stack Overflow, using the tags *pharo>http://stackoverflow.com/questions/tagged/pharo* and *smalltalk>http://stackoverflow.com/questions/tagged/smalltalk*.
-- SmalltalkHub (*http://www.smalltalkhub.com/>http://www.smalltalkhub.com/*)
-is the equivalent of SourceForge/Github for Pharo projects. Many extra packages
-and projects for Pharo live there.
+- SmalltalkHub (*http://www.smalltalkhub.com/>http://www.smalltalkhub.com/*) is the equivalent of SourceForge/Github for Pharo projects. Many extra packages and projects for Pharo live there.
 
 !!Examples and exercises
 
 We have tried to provide as many examples as possible. In particular, there are
 many examples that show a fragment of code which can be evaluated. We use the
-symbol ==-->== an expression and to indicate the result that you obtain when you
-select an expression and ""print it"":
+symbol {{{latex:$\to$}}}{{{html:<code>--&gt;</code>}}} to indicate the result 
```

Comment:
We must keep `-->` everywhere. Yes, it is less nice, but this is what will appear inside code blocks